src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/Maintainability/UseCrossPlatformIntrinsics.Fixer.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.Operations;
+
+namespace Microsoft.CodeQuality.Analyzers.Maintainability
+{
+    using static MicrosoftCodeQualityAnalyzersResources;
+    using RuleKind = UseCrossPlatformIntrinsicsAnalyzer.RuleKind;
+
+    public abstract class UseCrossPlatformIntrinsicsFixer : CodeFixProvider
+    {
+        public sealed override ImmutableArray<string> FixableDiagnosticIds { get; } = ImmutableArray.Create(
+            UseCrossPlatformIntrinsicsAnalyzer.RuleId
+        );
```

Comment:
```suggestion
        public sealed override ImmutableArray<string> FixableDiagnosticIds { get; } = [UseCrossPlatformIntrinsicsAnalyzer.RuleId];
```

Reply:
This can't be done because `ImmutableArray` cannot be used with collection expressions downlevel. We report `CS9210`.